user/languages/matlab.md

```diff
@@ -35,10 +36,6 @@ This guide covers build environment and configuration topics specific to
 > Currently, MATLAB builds are available only for public projects in Linux&reg; environments.
 
 
-### Community-Supported Language
-
-The MATLAB language is maintained by MathWorks&reg;. If you have any questions or suggestions, please contact MathWorks at [continuous-integration@mathworks.com](mailto:continuous-integration@mathworks.com).
-
 ## Specify MATLAB Releases and Run Tests
 
 Specify MATLAB releases using the `matlab` key. You can specify R2020a or a later release. If you do not specify a release, Travis CI uses the latest release of MATLAB. 
```

Comment:
`latest` will just stay as R2024a indefinitely. I want to keep the scope of the changes in this PR small if possible so I would prefer to avoid changing grammar or examples. Maybe that can be done in another PR before merging if we decide to do so.